SmartSaver.Domain/Repositories/Transaction/ITransactionRepo.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SmartSaver.EntityFrameworkCore.Models;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SmartSaver.Domain.Repositories
+{
+    public enum CreateTransactionResponse
+    {
+        Success,
+        BadAmountError,
+        GeneralError
+    }
+
+    public interface ITransactionRepo
+    {
+        /// <summary>
+        /// Method that returns all account transactions from spending category.
+        /// </summary>
+        /// <param name="accId"></param>
+        /// <returns>Spendings</returns>
```

Comment:
It is clear what method is doing when you look at its' name. So comments are redundant. It will make readability better if they be removed. This comment is applied to all methods in this interface